File: 1064 - Positivos e média.c
#include <stdio.h>

int main(void) {
  
  double n, acumulador;
  int cont, contn = 0;

  for (cont = 1; cont <= 6; cont++){

    scanf(" %lf", &n);

    if (n > 0){
      contn++;
      acumulador += (double)n;  
    }
  }
  printf("%d valores positivos\n", contn);
  printf("%.1f\n", acumulador/(double)contn);

  return 0;
}

File: Python/Lista 4/1064 - Positivos e Média.py
c = 0
acumulador = 0

for i in range(0, 6):
    n = float(input())
    if n > 0:
        c += 1
        acumulador += n

media = acumulador / c

print("{} valores positivos".format(c))
print("{:.1f}".format(media))
